src/opera/test/pge/dswx_hls/test_dswx_hls_pge.py

```diff
@@ -20,13 +20,13 @@
 
 import yaml
 
-import opera.util.img_utils
+import opera.util.tiff_utils
 from opera.pge import RunConfig
 from opera.pge.dswx_hls.dswx_hls_pge import DSWxHLSExecutor
 from opera.util import PgeLogger
-from opera.util.img_utils import MockGdal
-from opera.util.metadata_utils import get_geographic_boundaries_from_mgrs_tile
-from opera.util.metadata_utils import get_sensor_from_spacecraft_name
+from opera.util.dataset_utils import get_sensor_from_spacecraft_name
+from opera.util.geo_utils import get_geographic_boundaries_from_mgrs_tile
+from opera.util.tiff_utils import MockGdal
```

Comment:
Another instance where we should be importing `MockGdal` directly from`mock_utils`

Reply:
Done.